silverstrike/views/accounts.py

```diff
@@ -154,6 +155,41 @@ def get_context_data(self, **kwargs):
         self.dstart = self.dstart or first_date
         context['dstart'] = self.dstart
         context['dend'] = self.dend
+        today = date.today()
+        one_month_ago = today - relativedelta(months=1)
+        two_months_ago = today - relativedelta(months=2)
+        context['date_ranges'] = [
```

Comment:
I think is a matter of choice.
I like it better in the backend because we make the html smaller. But I agree that we should send the date we need and parse it in the html